ui/StatusQ/include/StatusQ/sumaggregator.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <QAbstractListModel>
+#include <QObject>
+#include <QVariant>
+#include <QDebug>
+
+#include "StatusQ/aggregator.h"
+
+class SumAggregator : public Aggregator {
+
+public:
+    explicit SumAggregator(QObject *parent = nullptr) {}
+
+private slots:
+    virtual void calculateAggregation() override {
```

Comment:
Ah there's no CPP file here :) OK then... but it would be better to have it instead of having the implementation in a header file

Reply:
OK! I will create the cpp file. I just thought that with just one implementation method would be fine in the `.h`, but it can potentially grow so... makes sense!